src/Task1Lesson3.java

```diff
@@ -0,0 +1,23 @@
+public class Task1Lesson3 {
+    //1. Given a string. Write a code that will create a new string made of the first two characters of the initial string.
+    //If the string is shorter than length 2, return whatever there is, so "X" yields "X", and the empty string "" yields
+    // the empty string "".
+    //
+    //For example:
+    //"AE_1381" -> "AE"
+    //"A" -> "A"
+    //"" -> ""
+    public static void main(String[] args){
+        String example = "AE_1381";
+        String s = example.substring(0, 2);
+        System.out.println(s);
+        String a = "A";
+        String c = "";
+        // String c for isEmpty
+        System.out.println(c.isEmpty());
+
+        if (a.length() <= 2){
+            System.out.println("String " +a);
+        }
```

Comment:
Your code should look like:

    public static void main(String[] args) {
        System.out.println(getStr("AE_1381"));
        System.out.println(getStr("A"));
        System.out.println(getStr(""));
    }


    public static String getStr(String str) {

    }